services/database/vector/threats.yaml

```diff
@@ -0,0 +1,129 @@
+    description: Attackers may infer sensitive information through metadata filters
+      or by correlating embeddings across modalities (e.g., voice and face), bypassing
+      surface-level access controls.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F04
+          - CCC.Vector.F10
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-RC-022
+  - id: CCC.Vector.TH04
+    title: Denial of Service via Query or Ingestion Flooding
+    description: High-volume ingestion or excessive low-latency similarity queries can
+      overwhelm vector indexing services, degrading performance for legitimate users
+      and potentially exhausting system resources.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F05
+          - CCC.Vector.F06
+    external-mappings:
+      - reference-id: MITRE-ATT&CK
+        identifiers:
+          - T1499 # Endpoint Denial of Service
+          - T1498 # Network Denial of Service
+          - T1496 # Resource Hijacking
+  - id: CCC.Vector.TH05
+    title: Index Corruption or Downgrade
+    description: Attackers with unauthorized access or excessive permissions may tamper
+      with or roll back index versions, potentially restoring poisoned data or breaking
+      downstream integrations.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F07
+          - CCC.Vector.F11
+    external-mappings:
+      - reference-id: MITRE-ATT&CK
+        identifiers:
+          - T1565 # Data Manipulation
+          - T1485 # Data Destruction
+          - T1490 # Inhibit System Recovery
+          - T1562 # Impair Defenses
+  - id: CCC.Vector.TH06
+    title: Embedding Format or Dimension Attacks
+    description: Poor validation of embedding formats or dimensions can cause service
+      crashes or logic errors. This can result in denial of service or incorrect similarity
+      results.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F08
+          - CCC.Vector.F09
+    external-mappings:
+      - reference-id: MITRE-ATT&CK
+        identifiers:
+          - T1499 # Endpoint Denial of Service
+          - T1565 # Data Manipulation
+          - T1027 # Obfuscated Files or Information
+  - id: CCC.Vector.TH07
+    title: Search Result Manipulation via ANN Bias
+    description: Approximate nearest neighbor (ANN) algorithms may yield non-deterministic
+      or biased results. Adversaries may exploit these differences to evade detection
+      or bias AI responses.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F12
+    external-mappings:
+      - reference-id: MITRE-ATT&CK
+        identifiers:
+          - T1565 # Data Manipulation
+          - T1027 # Obfuscated Files or Information
+          - T1497 # Virtualization/Sandbox Evasion
```

Comment:
`T1497` refers more to execution sandboxes and malware, e.g. an implant changing behaviour if it detects it's in a sandbox to avoid analysis